src/packages/cli/src/detach.ts

```diff
@@ -19,6 +19,8 @@ export type DetachedInstance = {
   version: string;
 };
 
+const MAX_SUGGESTIONS = 4;
+const MAX_LEVENSHTEIN_DISTANCE = 10;
```

Comment:
This is a _huge_ distance. I'm unsure about how to tune this. 

I'd rather we give false positives, than false negatives - if I _forget_ the name of the instance, and get something that's truly wrong, I'd still like a recommendation.

I'm open to suggestions on this. At the same time, it's something that we can tune later, and hopefully we'll get some user feedback on the experience.